eng/Tools.props

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Where is this file referenced?

Reply:
Can you make sure to add a comment to the file indicating this?  It's content looks a bit unusual for a props file and it'd be good to specify what its purpose is especially since it's hooking into an external project.